data/mods/BombasticPerks/perkdata/firstest_aid.json

```
@@ -0,0 +1,19 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_PERK_FIRSTEST_AID",
+    "eoc_type": "EVENT",
+    "required_event": "character_takes_damage",
+    "condition": { "u_has_trait": "perk_firstest_aid" },
+    "effect": [ 
+      { "u_lose_effect": "effect_perk_firstest_aid" }, 
+      { "u_add_effect": "effect_perk_firstest_aid", "duration": "2 minutes" } ]
+  },
+  {
+    "type": "effect_type",
+    "id": "effect_perk_firstest_aid",
+    "//": "Hidden effect used as a tracker",
```

```suggestion
    "//": "Hidden effect used as a tracker",
    "max_duration": "2 m",
```
we don't wanna player to stack the buff now don't we?

---

It shouldn't be stackable since the EoC removes it before reapplying it, but better to be cautious.